src/Interpreter.test.ts

```diff
@@ -611,7 +605,7 @@ test('interpret empty ballot', async () => {
   await expect(
     interpreter.interpretBallot(await blankPage1.imageData())
   ).rejects.toThrow(
-    'Refusing to interpret ballots before all templates are added.'
+    'Cannot scan ballot because not all required templates have been added'
```

Comment:
based on my failed debugging, it would be nice if this listed at least one of the missing templates.

Reply:
oh never mind I see that this happens now in `debug()` statements below :)